Group-3/src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -16,46 +16,46 @@
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Intake extends SubsystemBase {
-  /** Creates a new ExampleSubsystem. */
+  /** Creates a new Intake Subsystem. */
 
+  // The compressor for the pneumatics 
   private Compressor pcmCompressor = new Compressor(Constants.Intake.compressor_id, PneumaticsModuleType.CTREPCM);
 
+  // the two solidoids that manage the movement of the arm
   private DoubleSolenoid leftDoubleSolenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.Intake.left_solenoid_forward_channel, Constants.Intake.left_solenoid_reverse_channel);
   private DoubleSolenoid rightDoubleSolenoid = new DoubleSolenoid(PneumaticsModuleType.CTREPCM, Constants.Intake.right_solenoid_forward_channel, Constants.Intake.right_solenoid_reverse_channel);
+  // the motor used to spin the motor on the arm
```

Comment:
Comments above the motor controller declarations are unnecessary, but you won't need to make any changes